### PR TITLE
adding new method for windows sandbox detection used by QBOT

### PR DESCRIPTION
**Pull request by emtuls**

### CODE REVIEW

**Review by williballenthin — APPROVED** [review]
logic looks good; lets see how CI/linter goes

File: anti-analysis/anti-vm/vm-detection/check-for-windows-sandbox-via-subdirectory.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - "echernofsky@google.com"
+    scope: basic block
+    att&ck:
+      - Defense Evasion::Virtualization/Sandbox Evasion::System Checks [T1497.001]
+    mbc:
+      - Anti-Behavioral Analysis::Virtual Machine Detection [B0009]
+    references:
+      - https://www.elastic.co/security-labs/qbot-malware-analysis
+    examples:
+      - ffaf9cd085434fd5230511ab895ba494:0x10015A6A
```
Comment (mr-tz) [inline anti-analysis/anti-vm/vm-detection/check-for-windows-sandbox-via-subdirectory.yml:15]: if this is the according testfile, let's change to use sha256 sum here
https://github.com/mandiant/capa-testfiles/pull/166

**Comment by mr-tz** [comment]
Lint on the sample seems to fail: `FAIL: doesn't match on referenced example: Fix the rule logic or provide a different example`. Does it work locally?

**Review by mr-tz — APPROVED** [review]
thanks!

**Pull request merged**